inc/umock_c_internal.h

```diff
@@ -572,10 +572,27 @@ typedef int(*TRACK_DESTROY_FUNC_TYPE)(PAIRED_HANDLES* paired_handles, const void
         return mock_call_modifier; \
     } \
 
+/* Codes_SRS_UMOCK_C_LIB_01_101: [The IgnoreAllCalls call modifier shall record that all calls matching the expected call shall be ignored. If no matching call occurs no missing call shall be reported.]*/
+/* Codes_SRS_UMOCK_C_LIB_01_208: [ If no matching call occurs no missing call shall be reported. ]*/
 #define IMPLEMENT_IGNORE_ALL_CALLS_FUNCTION(return_type, name, ...) \
     static C2(mock_call_modifier_,name) C2(ignore_all_calls_func_,name)(void) \
     { \
         DECLARE_MOCK_CALL_MODIFIER(name) \
+        UMOCKCALL_HANDLE last_expected_call = umock_c_get_last_expected_call(); \
```

Comment:
Added.

---
In reply to: [106217765](https://github.com/Azure/umock-c/pull/2#discussion_r106217765) [](ancestors = 106217765)